Project.toml

```diff
@@ -18,7 +18,7 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 [compat]
 Adapt = "3.3"
 CEnum = "0.4"
-Enzyme_jll = "0.0.40"
+Enzyme_jll = "0.0.39"
```

Comment:
Was this a test? Or a intentional downgrade?

Reply:
Didnt intend to add Project.toml, just locally hadn't propagated and was lazy.